src/scratchtocatrobat/converter/converter.py

```diff
@@ -1422,4 +1427,18 @@ def _convert_answer_block(self):
                                                                                           data_container,
                                                                                           self.script_context)
         return _variable_for(shared_global_answer_user_variable.getName())
+    
+    @_register_handler(_block_name_to_handler_map, "createCloneOf")
+    def _convert_create_clone_of_block(self):
+        [base_sprite] = self.arguments
+        if isinstance(base_sprite, catformula.FormulaElement):
+            return catbricks.NoteBrick("Can't convert Clone-Brick with Formula as argument.\n"\
+                                       "This is not supported in Pocket Code.")
```

Comment:
Please remove "This is not supported ..." from the message as the sentence before should already be self-explanatory.